[CTOR-1171] Refresh hardware-servers-ibm-bladecenter-snmp doc page (#3946)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-servers-ibm-bladecenter-snmp.md

id: hardware-servers-ibm-bladecenter-snmp
title: IBM BladeCenter
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **IBM BladeCenter** connector through the
**Configuration > Monitoring Connector Manager** menu:
* [Base Pack](./base-generic.md)

## Pack assets

### Templates

The Monitoring Connector **IBM BladeCenter** brings a host template:

* **HW-Server-IBM-Bladecenter-SNMP-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="HW-Server-IBM-Bladecenter-SNMP-custom" label="HW-Server-IBM-Bladecenter-SNMP-custom">

| Service Alias   | Service Template                               | Service Description                                                                                                    |
|:----------------|:-----------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------|

> | Hardware-Global | HW-IBM-Bladecenter-Hardware-Global-SNMP-custom | Check hardware (ambient temperatures, blowers, power modules, system health, chassis tests) of IBM Bladecenter chassis |

Reviewer: ```suggestion
| Hardware-Global | HW-IBM-Bladecenter-Hardware-Global-SNMP-custom | Check the hardware components of an IBM Bladecenter chassis |
```

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias           | Service Template                                       | Service Description                                              |
|:------------------------|:-------------------------------------------------------|:-----------------------------------------------------------------|
| Hardware-Ambient        | HW-IBM-Bladecenter-Hardware-Ambient-SNMP-custom        | Check 'ambient temperatures' hardware of IBM Bladecenter chassis |
| Hardware-Blade          | HW-IBM-Bladecenter-Hardware-Blade-SNMP-custom          | Check 'blades' hardware of IBM Bladecenter chassis               |
| Hardware-Blower         | HW-IBM-Bladecenter-Hardware-Blower-SNMP-custom         | Check 'blowers' hardware of IBM Bladecenter chassis              |
| Hardware-Chassis-Status | HW-IBM-Bladecenter-Hardware-Chassis-Status-SNMP-custom | Check hardware tests of IBM Bladecenter chassis                  |
| Hardware-Power-Module   | HW-IBM-Bladecenter-Hardware-Power-Module-SNMP-custom   | Check 'power modules' hardware of IBM Bladecenter chassis        |

> | Hardware-System-Health  | HW-IBM-Bladecenter-Hardware-System-Health-SNMP-custom  | Check 'system health' hardware of IBM Bladecenter chassis        |

Reviewer: hardware everywhere?

</TabItem>
<TabItem value="Hardware-System-Health" label="Hardware-System-Health">

| Macro        | Description                                                                                                                                                               | Default value     | Mandatory   |
|:-------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| COMPONENT    | Which component to check (default: 'all'). Can be: 'ambient', 'powermodule', 'fanpack', 'chassisfan',  'blower', 'blade', 'systemhealth', 'chassisstatus', 'switchmodule' | systemhealth      |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                                        | --verbose         |             |

> 

Reviewer: default?
